<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-LU9FG0N5" failures="5" tests="5" name="Add Customer Login Test" time="13.684" errors="0" timestamp="2021-05-05T18:05:34 EDT">
  <testcase classname="testcases.AddCustomerTest" name="addCustomerTest" time="2.997">
    <failure type="java.lang.AssertionError" message="Customer not added Successfull">
      <![CDATA[java.lang.AssertionError: Customer not added Successfull
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase classname="testcases.AddCustomerTest" name="addCustomerTest" time="2.306">
    <failure type="java.lang.AssertionError" message="Customer not added Successfull">
      <![CDATA[java.lang.AssertionError: Customer not added Successfull
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase classname="testcases.AddCustomerTest" name="addCustomerTest" time="2.319">
    <failure type="java.lang.AssertionError" message="Customer not added Successfull">
      <![CDATA[java.lang.AssertionError: Customer not added Successfull
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase classname="testcases.AddCustomerTest" name="addCustomerTest" time="2.369">
    <failure type="java.lang.AssertionError" message="Customer not added Successfull">
      <![CDATA[java.lang.AssertionError: Customer not added Successfull
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase classname="testcases.AddCustomerTest" name="addCustomerTest" time="2.286">
    <failure type="java.lang.AssertionError" message="Customer not added Successfull">
      <![CDATA[java.lang.AssertionError: Customer not added Successfull
at testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
</testsuite> <!-- Add Customer Login Test -->
